main.tf

```diff
@@ -190,6 +200,16 @@ resource "aws_cloudfront_distribution" "this" {
           include_body = lookup(l.value, "include_body", null)
         }
       }
+
+      dynamic "function_association" {
+        for_each = lookup(i.value, "function_association", [])
```

Comment:
The default value for `lookup` should be an empty object/map `{}`

Reply:
It can be but it is not a requirement or a bug. Both options are valid here.

Reply:
yeah, I opted for `[]` as per the block above it for the lambdas, to keep it consistent